content/kubeone/master/guides/addons/_index.en.md

```diff
@@ -81,9 +85,105 @@ the path is relative to the KubeOne configuration file. This means that
 
 {{% notice note %}}
 Subdirectories are not considered when applying addons. Only addons in the root
-of the provided directory will be applied.
+of the provided directory will be applied. E.g. `./addons/addon_name1` is
+supported addon while `./addons/groupping/addon_name1` and
+`./addons/addon_name1/subdir1` does not.
+{{% /notice %}}
+
+## Embedded Addons
+
+Since v1.3 release KubeOne has new kind of addons that always caried with the
+binariy itself, embedded addons. This is done thanks to new Go's
+[`embed`][embed-docs] package. 
+
+Some of those addons are directly and automatically used by the KubeOne itself
+(like machinecontroller for example). But it's not the case for all of them.
+Some very useful addons need to be activated by the user in order to be deployed
+to the cluster, like `backups-restic`, `cluster-autoscaler` or
+`unattended-upgrades`.
+
+### Activate Embedded Eddons
+
+Or order to activate such embedded addons, user need to use new Addons
+[API][addons-api].
+
+Example:
+```yaml
+apiVersion: kubeone.io/v1beta1
+kind: KubeOneCluster
+versions:
+  kubernetes: 1.20.1
+
+addons:
+  enable: true
+  path: "./addons"
+  addons:
+  - name: cluster-autoscaler
+  - name: unattended-upgrades
+```
+
+[Up to date (and possibly unreleased yet) list of addons.][addons-list-url]
```

Comment:
I'd remove `(and possibly unreleased yet)` because we might very easily forget to remove this when releasing the stable version of docs. Someone browsing the master version of the docs should be aware that many things might be unreleased. Moreover, there's a warning at the top:

> This version is under construction, please use an official release version

```suggestion
[Up to date list of addons is available on GitHub.][addons-list-url]
```

Reply:
Additionally, users might not know which addons are deployed by default and which are provided to be used with the Addons API. We should maintain a complete list somewhere, probably both on GitHub and docs.

Reply:
The phase `(and possibly unreleased yet)` is a look past 1.3 release, in the future. When we will add some new addons in the future, there will a time gap in between a releases which falls right under "not released yet". This phase is not be to deleted after the release.